maintenance/wikia/sql/update-schemas.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+DATABASES='dataware specials stats swift_sync wikicities'
+
+for DATABASE in $DATABASES
+do
+	echo "Updating schema of tables in $DATABASE database ..."
+
+	DB_PARAMS=`dbparams.pl --type slave --name $DATABASE`
+	mysqldump --set-gtid-purged=OFF --no-data $DB_PARAMS | grep -vE '^\/\*\!' | sed 's/AUTO_INCREMENT=[0-9]* //g' > ${DATABASE}-schema.sql
```

Comment:
I suggest using long options consistently, unless we want to run the script just once.